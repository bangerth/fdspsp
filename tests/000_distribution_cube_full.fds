!!
!! Copyright (c) 2020 by the FireDynamics authors
!!
!! This file is part of the FDS particle spray postprocessor (fdspsp).
!!
!! fdspsp is free software; you can use it, redistribute it, and/or
!! modify it under the terms of the MIT License. The full text of the
!! license can be found in the file LICENSE.md at the top level
!! directory of fdspsp.
!!


&HEAD CHID='000_distribution_cube_full',
      TITLE='000: uniform particle distribution in an entire cube' /

&TIME T_END=1., DT=1., LOCK_TIME_STEP=.TRUE. /

&MESH ID='1' IJK=3,3,3, XB=0.0,1.0,0.0,1.0,0.0,1.0 /
&MESH ID='2' IJK=3,3,3, XB=1.0,2.0,0.0,1.0,0.0,1.0 /
&MESH ID='3' IJK=3,3,3, XB=0.0,1.0,1.0,2.0,0.0,1.0 /
&MESH ID='4' IJK=3,3,3, XB=1.0,2.0,1.0,2.0,0.0,1.0 /
&MESH ID='5' IJK=3,3,3, XB=0.0,1.0,0.0,1.0,1.0,2.0 /
&MESH ID='6' IJK=3,3,3, XB=1.0,2.0,0.0,1.0,1.0,2.0 /
&MESH ID='7' IJK=3,3,3, XB=0.0,1.0,1.0,2.0,1.0,2.0 /
&MESH ID='8' IJK=3,3,3, XB=1.0,2.0,1.0,2.0,1.0,2.0 /


&PART ID='tracer', MASSLESS=.TRUE., SAMPLING_FACTOR=1,
      QUANTITIES(1:7)='PARTICLE DIAMETER', 'PARTICLE MASS', 'PARTICLE TEMPERATURE', 'PARTICLE U', 'PARTICLE V', 'PARTICLE W', 'PARTICLE WEIGHTING FACTOR' /

&INIT PART_ID='tracer', XB=0.0,2.0,0.0,2.0,0.0,2.0, N_PARTICLES_PER_CELL=10 /

&TAIL /
