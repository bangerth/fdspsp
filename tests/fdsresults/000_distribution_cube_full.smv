TITLE
 000: uniform particle distribution in an entire cube

FDSVERSION
FDS6.7.5-318-gac3275a6c-master

ENDF
 000_distribution_cube_full.end

INPF
 /home/marc/work/codes/fds/fdspsp/tests/000_distribution_cube_full.fds

REVISION
Revisio                                                                                                                                                                                                                                                        

CHID
 000_distribution_cube_full

SOLID_HT3D
  0

CSVF
 hrr
 000_distribution_cube_full_hrr.csv

CSVF
 steps
 000_distribution_cube_full_steps.csv

NMESHES
  8

VIEWTIMES
      0.00      1.00  1000

ALBEDO
      0.30000

IBLANK
 1

GVEC
      0.00000      0.00000     -9.80665

SURFDEF
 INERT                                                       

SURFACE
 INERT                                                       
 5000.00    1.00
 0      1.00000      1.00000      1.00000      0.80000      0.40000      1.00000
 null                                                        

SURFACE
 OPEN                                                        
 5000.00    1.00
 2      1.00000      1.00000      1.00000      0.00000      1.00000      1.00000
 null                                                        

SURFACE
 MIRROR                                                      
 5000.00    1.00
-2      1.00000      1.00000      1.00000      0.80000      0.40000      1.00000
 null                                                        

SURFACE
 INTERPOLATED                                                
 5000.00    1.00
 0      1.00000      1.00000      1.00000      0.80000      0.40000      1.00000
 null                                                        

SURFACE
 PERIODIC                                                    
 5000.00    1.00
 0      1.00000      1.00000      1.00000      0.80000      0.40000      1.00000
 null                                                        

SURFACE
 HVAC                                                        
 5000.00    1.00
 0      1.00000      1.00000      1.00000      0.80000      0.40000      1.00000
 null                                                        

SURFACE
 MASSLESS TRACER                                             
 5000.00    1.00
 0      1.00000      1.00000      1.00000      0.80000      0.40000      1.00000
 null                                                        

SURFACE
 DROPLET                                                     
 5000.00    1.00
 0      1.00000      1.00000      1.00000      0.80000      0.40000      1.00000
 null                                                        

SURFACE
 EVACUATION_OUTFLOW                                          
 5000.00    1.00
 0      1.00000      1.00000      1.00000      0.80000      0.40000      1.00000
 null                                                        

SURFACE
 MASSLESS TARGET                                             
 5000.00    1.00
 0      1.00000      1.00000      1.00000      0.80000      0.40000      1.00000
 null                                                        

SURFACE
 PERIODIC FLOW ONLY                                          
 5000.00    1.00
 0      1.00000      1.00000      1.00000      0.80000      0.40000      1.00000
 null                                                        

CLASS_OF_PARTICLES
 tracer                                                      
      0.00000      0.00000      0.00000
  7
 PARTICLE DIAMETER                                           
 diam                                                        
 mu-m                                                        
 PARTICLE MASS                                               
 mass                                                        
 kg                                                          
 PARTICLE TEMPERATURE                                        
 temp                                                        
 C                                                           
 PARTICLE U                                                  
 u                                                           
 m/s                                                         
 PARTICLE V                                                  
 v                                                           
 m/s                                                         
 PARTICLE W                                                  
 w                                                           
 m/s                                                         
 PARTICLE WEIGHTING FACTOR                                   
 pwt                                                         
                                                             

CLASS_OF_PARTICLES
 tracer_noquantities                                         
      0.00000      0.00000      0.00000
  0

OUTLINE
  24
        0.0000        0.0000        0.0000        0.0000        0.0000        1.0000
        2.0000        0.0000        0.0000        2.0000        0.0000        1.0000
        0.0000        2.0000        0.0000        0.0000        2.0000        1.0000
        2.0000        2.0000        0.0000        2.0000        2.0000        1.0000
        0.0000        0.0000        1.0000        0.0000        0.0000        2.0000
        2.0000        0.0000        1.0000        2.0000        0.0000        2.0000
        0.0000        2.0000        1.0000        0.0000        2.0000        2.0000
        2.0000        2.0000        1.0000        2.0000        2.0000        2.0000
        0.0000        0.0000        0.0000        0.0000        1.0000        0.0000
        2.0000        0.0000        0.0000        2.0000        1.0000        0.0000
        0.0000        1.0000        0.0000        0.0000        2.0000        0.0000
        2.0000        1.0000        0.0000        2.0000        2.0000        0.0000
        0.0000        0.0000        2.0000        0.0000        1.0000        2.0000
        2.0000        0.0000        2.0000        2.0000        1.0000        2.0000
        0.0000        1.0000        2.0000        0.0000        2.0000        2.0000
        2.0000        1.0000        2.0000        2.0000        2.0000        2.0000
        0.0000        0.0000        0.0000        1.0000        0.0000        0.0000
        1.0000        0.0000        0.0000        2.0000        0.0000        0.0000
        0.0000        2.0000        0.0000        1.0000        2.0000        0.0000
        1.0000        2.0000        0.0000        2.0000        2.0000        0.0000
        0.0000        0.0000        2.0000        1.0000        0.0000        2.0000
        1.0000        0.0000        2.0000        2.0000        0.0000        2.0000
        0.0000        2.0000        2.0000        1.0000        2.0000        2.0000
        1.0000        2.0000        2.0000        2.0000        2.0000        2.0000

TOFFSET
      0.00000      0.00000      0.00000

HRRPUVCUT
     1
     60.00000

RAMP
    2
 RAMP: RSRVD TEMPERATURE PROFILE                                   
    2
  0.0000      1.0000    
  2.0000      1.0000    
 RAMP: RSRVD PRESSURE PROFILE                                      
    2
  0.0000      0.0000    
  2.0000      1.0000    

PROP
 null
  1
 sensor
  0

OFFSET
      0.00000      0.00000      0.00000

GRID   1
    3    3    3    0

PDIM
       0.00000       1.00000       0.00000       1.00000       0.00000       1.00000       0.00000       0.00000       0.00000

TRNX
    0
    0       0.00000
    1       0.33333
    2       0.66667
    3       1.00000

TRNY
    0
    0       0.00000
    1       0.33333
    2       0.66667
    3       1.00000

TRNZ
    0
    0       0.00000
    1       0.33333
    2       0.66667
    3       1.00000

OBST
           0

VENT
    3    3
       0.00000       0.00000       0.00000       1.00000       0.00000       1.00000     1   0
       0.00000       1.00000       0.00000       0.00000       0.00000       1.00000     2   0
       0.00000       1.00000       0.00000       1.00000       0.00000       0.00000     3   0
    0    0    0    3    0    3   99    0
    0    3    0    0    0    3   99    0
    0    3    0    3    0    0   99    0

CVENT
    0


OFFSET
      0.00000      0.00000      0.00000

GRID   2
    3    3    3    0

PDIM
       1.00000       2.00000       0.00000       1.00000       0.00000       1.00000       0.00000       0.00000       0.00000

TRNX
    0
    0       1.00000
    1       1.33333
    2       1.66667
    3       2.00000

TRNY
    0
    0       0.00000
    1       0.33333
    2       0.66667
    3       1.00000

TRNZ
    0
    0       0.00000
    1       0.33333
    2       0.66667
    3       1.00000

OBST
           0

VENT
    3    3
       2.00000       2.00000       0.00000       1.00000       0.00000       1.00000     1   0
       1.00000       2.00000       0.00000       0.00000       0.00000       1.00000     2   0
       1.00000       2.00000       0.00000       1.00000       0.00000       0.00000     3   0
    3    3    0    3    0    3   99    0
    0    3    0    0    0    3   99    0
    0    3    0    3    0    0   99    0

CVENT
    0


OFFSET
      0.00000      0.00000      0.00000

GRID   3
    3    3    3    0

PDIM
       0.00000       1.00000       1.00000       2.00000       0.00000       1.00000       0.00000       0.00000       0.00000

TRNX
    0
    0       0.00000
    1       0.33333
    2       0.66667
    3       1.00000

TRNY
    0
    0       1.00000
    1       1.33333
    2       1.66667
    3       2.00000

TRNZ
    0
    0       0.00000
    1       0.33333
    2       0.66667
    3       1.00000

OBST
           0

VENT
    3    3
       0.00000       0.00000       1.00000       2.00000       0.00000       1.00000     1   0
       0.00000       1.00000       2.00000       2.00000       0.00000       1.00000     2   0
       0.00000       1.00000       1.00000       2.00000       0.00000       0.00000     3   0
    0    0    0    3    0    3   99    0
    0    3    3    3    0    3   99    0
    0    3    0    3    0    0   99    0

CVENT
    0


OFFSET
      0.00000      0.00000      0.00000

GRID   4
    3    3    3    0

PDIM
       1.00000       2.00000       1.00000       2.00000       0.00000       1.00000       0.00000       0.00000       0.00000

TRNX
    0
    0       1.00000
    1       1.33333
    2       1.66667
    3       2.00000

TRNY
    0
    0       1.00000
    1       1.33333
    2       1.66667
    3       2.00000

TRNZ
    0
    0       0.00000
    1       0.33333
    2       0.66667
    3       1.00000

OBST
           0

VENT
    3    3
       2.00000       2.00000       1.00000       2.00000       0.00000       1.00000     1   0
       1.00000       2.00000       2.00000       2.00000       0.00000       1.00000     2   0
       1.00000       2.00000       1.00000       2.00000       0.00000       0.00000     3   0
    3    3    0    3    0    3   99    0
    0    3    3    3    0    3   99    0
    0    3    0    3    0    0   99    0

CVENT
    0


OFFSET
      0.00000      0.00000      0.00000

GRID   5
    3    3    3    0

PDIM
       0.00000       1.00000       0.00000       1.00000       1.00000       2.00000       0.00000       0.00000       0.00000

TRNX
    0
    0       0.00000
    1       0.33333
    2       0.66667
    3       1.00000

TRNY
    0
    0       0.00000
    1       0.33333
    2       0.66667
    3       1.00000

TRNZ
    0
    0       1.00000
    1       1.33333
    2       1.66667
    3       2.00000

OBST
           0

VENT
    3    3
       0.00000       0.00000       0.00000       1.00000       1.00000       2.00000     1   0
       0.00000       1.00000       0.00000       0.00000       1.00000       2.00000     2   0
       0.00000       1.00000       0.00000       1.00000       2.00000       2.00000     3   0
    0    0    0    3    0    3   99    0
    0    3    0    0    0    3   99    0
    0    3    0    3    3    3   99    0

CVENT
    0


OFFSET
      0.00000      0.00000      0.00000

GRID   6
    3    3    3    0

PDIM
       1.00000       2.00000       0.00000       1.00000       1.00000       2.00000       0.00000       0.00000       0.00000

TRNX
    0
    0       1.00000
    1       1.33333
    2       1.66667
    3       2.00000

TRNY
    0
    0       0.00000
    1       0.33333
    2       0.66667
    3       1.00000

TRNZ
    0
    0       1.00000
    1       1.33333
    2       1.66667
    3       2.00000

OBST
           0

VENT
    3    3
       2.00000       2.00000       0.00000       1.00000       1.00000       2.00000     1   0
       1.00000       2.00000       0.00000       0.00000       1.00000       2.00000     2   0
       1.00000       2.00000       0.00000       1.00000       2.00000       2.00000     3   0
    3    3    0    3    0    3   99    0
    0    3    0    0    0    3   99    0
    0    3    0    3    3    3   99    0

CVENT
    0


OFFSET
      0.00000      0.00000      0.00000

GRID   7
    3    3    3    0

PDIM
       0.00000       1.00000       1.00000       2.00000       1.00000       2.00000       0.00000       0.00000       0.00000

TRNX
    0
    0       0.00000
    1       0.33333
    2       0.66667
    3       1.00000

TRNY
    0
    0       1.00000
    1       1.33333
    2       1.66667
    3       2.00000

TRNZ
    0
    0       1.00000
    1       1.33333
    2       1.66667
    3       2.00000

OBST
           0

VENT
    3    3
       0.00000       0.00000       1.00000       2.00000       1.00000       2.00000     1   0
       0.00000       1.00000       2.00000       2.00000       1.00000       2.00000     2   0
       0.00000       1.00000       1.00000       2.00000       2.00000       2.00000     3   0
    0    0    0    3    0    3   99    0
    0    3    3    3    0    3   99    0
    0    3    0    3    3    3   99    0

CVENT
    0


OFFSET
      0.00000      0.00000      0.00000

GRID   8
    3    3    3    0

PDIM
       1.00000       2.00000       1.00000       2.00000       1.00000       2.00000       0.00000       0.00000       0.00000

TRNX
    0
    0       1.00000
    1       1.33333
    2       1.66667
    3       2.00000

TRNY
    0
    0       1.00000
    1       1.33333
    2       1.66667
    3       2.00000

TRNZ
    0
    0       1.00000
    1       1.33333
    2       1.66667
    3       2.00000

OBST
           0

VENT
    3    3
       2.00000       2.00000       1.00000       2.00000       1.00000       2.00000     1   0
       1.00000       2.00000       2.00000       2.00000       1.00000       2.00000     2   0
       1.00000       2.00000       1.00000       2.00000       2.00000       2.00000     3   0
    3    3    0    3    0    3   99    0
    0    3    3    3    0    3   99    0
    0    3    0    3    3    3   99    0

CVENT
    0

SMOKF3D     1
 000_distribution_cube_full_0001_02.s3d
 HRRPUV
 hrrpuv
 kW/m3
SMOKG3D     1
 000_distribution_cube_full_0001_03.s3d
 TEMPERATURE
 temp
 C
SMOKG3D     1
 000_distribution_cube_full_0001_04.s3d
 CARBON DIOXIDE DENSITY
 rho_CO2
 kg/m3
PRT5     1
 000_distribution_cube_full_0001.prt5
      2
      1
      2
SMOKF3D     2
 000_distribution_cube_full_0002_02.s3d
 HRRPUV
 hrrpuv
 kW/m3
SMOKG3D     2
 000_distribution_cube_full_0002_03.s3d
 TEMPERATURE
 temp
 C
SMOKG3D     2
 000_distribution_cube_full_0002_04.s3d
 CARBON DIOXIDE DENSITY
 rho_CO2
 kg/m3
PRT5     2
 000_distribution_cube_full_0002.prt5
      2
      1
      2
SMOKF3D     3
 000_distribution_cube_full_0003_02.s3d
 HRRPUV
 hrrpuv
 kW/m3
SMOKG3D     3
 000_distribution_cube_full_0003_03.s3d
 TEMPERATURE
 temp
 C
SMOKG3D     3
 000_distribution_cube_full_0003_04.s3d
 CARBON DIOXIDE DENSITY
 rho_CO2
 kg/m3
PRT5     3
 000_distribution_cube_full_0003.prt5
      2
      1
      2
SMOKF3D     4
 000_distribution_cube_full_0004_02.s3d
 HRRPUV
 hrrpuv
 kW/m3
SMOKG3D     4
 000_distribution_cube_full_0004_03.s3d
 TEMPERATURE
 temp
 C
SMOKG3D     4
 000_distribution_cube_full_0004_04.s3d
 CARBON DIOXIDE DENSITY
 rho_CO2
 kg/m3
PRT5     4
 000_distribution_cube_full_0004.prt5
      2
      1
      2
SMOKF3D     5
 000_distribution_cube_full_0005_02.s3d
 HRRPUV
 hrrpuv
 kW/m3
SMOKG3D     5
 000_distribution_cube_full_0005_03.s3d
 TEMPERATURE
 temp
 C
SMOKG3D     5
 000_distribution_cube_full_0005_04.s3d
 CARBON DIOXIDE DENSITY
 rho_CO2
 kg/m3
PRT5     5
 000_distribution_cube_full_0005.prt5
      2
      1
      2
SMOKF3D     6
 000_distribution_cube_full_0006_02.s3d
 HRRPUV
 hrrpuv
 kW/m3
SMOKG3D     6
 000_distribution_cube_full_0006_03.s3d
 TEMPERATURE
 temp
 C
SMOKG3D     6
 000_distribution_cube_full_0006_04.s3d
 CARBON DIOXIDE DENSITY
 rho_CO2
 kg/m3
PRT5     6
 000_distribution_cube_full_0006.prt5
      2
      1
      2
SMOKF3D     7
 000_distribution_cube_full_0007_02.s3d
 HRRPUV
 hrrpuv
 kW/m3
SMOKG3D     7
 000_distribution_cube_full_0007_03.s3d
 TEMPERATURE
 temp
 C
SMOKG3D     7
 000_distribution_cube_full_0007_04.s3d
 CARBON DIOXIDE DENSITY
 rho_CO2
 kg/m3
PRT5     7
 000_distribution_cube_full_0007.prt5
      2
      1
      2
SMOKF3D     8
 000_distribution_cube_full_0008_02.s3d
 HRRPUV
 hrrpuv
 kW/m3
SMOKG3D     8
 000_distribution_cube_full_0008_03.s3d
 TEMPERATURE
 temp
 C
SMOKG3D     8
 000_distribution_cube_full_0008_04.s3d
 CARBON DIOXIDE DENSITY
 rho_CO2
 kg/m3
PRT5     8
 000_distribution_cube_full_0008.prt5
      2
      1
      2
