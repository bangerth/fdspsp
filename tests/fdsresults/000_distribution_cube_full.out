
 Fire Dynamics Simulator

 Current Date     : October 24, 2020  19:51:40
 Revision         : FDS6.7.5-318-gac3275a6c-master
 Revision Date    : Fri Oct 16 13:16:15 2020 -0400
 Compiler         : Gnu gfortran 10.2.0
 Compilation Date : Oct 17, 2020  00:20:10

 MPI Enabled;    Number of MPI Processes:       1
 OpenMP Enabled; Number of OpenMP Threads:      4

 MPI version: 3.1
 MPI library version: Open MPI v4.0.5, package: Open MPI builduser@anthraxx-slave-4 Distribution, ident: 4.0.5, repo rev: v4.0.5, Aug 26, 2020

 Job TITLE        : 000: uniform particle distribution in an entire cube
 Job ID string    : 000_distribution_cube_full


 Grid Dimensions, Mesh     1

   Cells in the X Direction             3
   Cells in the Y Direction             3
   Cells in the Z Direction             3
   Number of Grid Cells                27


 Physical Dimensions, Mesh     1

   Length (m)                       1.000
   Width  (m)                       1.000
   Height (m)                       1.000
   Initial Time Step (s)            1.000

 Grid Dimensions, Mesh     2

   Cells in the X Direction             3
   Cells in the Y Direction             3
   Cells in the Z Direction             3
   Number of Grid Cells                27


 Physical Dimensions, Mesh     2

   Length (m)                       1.000
   Width  (m)                       1.000
   Height (m)                       1.000
   Initial Time Step (s)            1.000

 Grid Dimensions, Mesh     3

   Cells in the X Direction             3
   Cells in the Y Direction             3
   Cells in the Z Direction             3
   Number of Grid Cells                27


 Physical Dimensions, Mesh     3

   Length (m)                       1.000
   Width  (m)                       1.000
   Height (m)                       1.000
   Initial Time Step (s)            1.000

 Grid Dimensions, Mesh     4

   Cells in the X Direction             3
   Cells in the Y Direction             3
   Cells in the Z Direction             3
   Number of Grid Cells                27


 Physical Dimensions, Mesh     4

   Length (m)                       1.000
   Width  (m)                       1.000
   Height (m)                       1.000
   Initial Time Step (s)            1.000

 Grid Dimensions, Mesh     5

   Cells in the X Direction             3
   Cells in the Y Direction             3
   Cells in the Z Direction             3
   Number of Grid Cells                27


 Physical Dimensions, Mesh     5

   Length (m)                       1.000
   Width  (m)                       1.000
   Height (m)                       1.000
   Initial Time Step (s)            1.000

 Grid Dimensions, Mesh     6

   Cells in the X Direction             3
   Cells in the Y Direction             3
   Cells in the Z Direction             3
   Number of Grid Cells                27


 Physical Dimensions, Mesh     6

   Length (m)                       1.000
   Width  (m)                       1.000
   Height (m)                       1.000
   Initial Time Step (s)            1.000

 Grid Dimensions, Mesh     7

   Cells in the X Direction             3
   Cells in the Y Direction             3
   Cells in the Z Direction             3
   Number of Grid Cells                27


 Physical Dimensions, Mesh     7

   Length (m)                       1.000
   Width  (m)                       1.000
   Height (m)                       1.000
   Initial Time Step (s)            1.000

 Grid Dimensions, Mesh     8

   Cells in the X Direction             3
   Cells in the Y Direction             3
   Cells in the Z Direction             3
   Number of Grid Cells                27


 Physical Dimensions, Mesh     8

   Length (m)                       1.000
   Width  (m)                       1.000
   Height (m)                       1.000
   Initial Time Step (s)            1.000

Total Number of Grid Cells           216


 Miscellaneous Parameters

   Simulation Start Time (s)          0.0
   Simulation End Time (s)            1.0
   LES Calculation
   Eddy Viscosity:           Deardorff Model (C_DEARDORFF = 0.10)
   Near-wall Eddy Viscosity: Smagorinsky with Van Driest damping (C_SMAGORINSKY = 0.20)
   Turbulent Prandtl Number:         0.50
   Turbulent Schmidt Number:         0.50
   Ambient Temperature (C):         20.00


 Mass Fraction Transformation Matrix to Convert Species Mixtures (Columns) to Primitive Species (Rows)

                         AIR     
   NITROGEN              0.762470
   OXYGEN                0.230997
   CARBON DIOXIDE        0.000591
   WATER VAPOR           0.005941


 Primitive Species Information

   NITROGEN
   Gas Species
   Molecular Weight (g/mol)            28.01340
   Ambient Density (kg/m^3)            1.165
   Enthalpy of Formation (J/kg)        0.00E+00

   OXYGEN
   Gas Species
   Molecular Weight (g/mol)            31.99880
   Ambient Density (kg/m^3)            1.330
   Enthalpy of Formation (J/kg)        0.00E+00

   CARBON DIOXIDE
   Gas Species
   Molecular Weight (g/mol)            44.00950
   Ambient Density (kg/m^3)            1.830
   Enthalpy of Formation (J/kg)       -8.94E+06

   WATER VAPOR
   Gas Species
   Molecular Weight (g/mol)            18.01528
   Ambient Density (kg/m^3)            0.749
   Enthalpy of Formation (J/kg)       -1.34E+07


 Tracked (Lumped) Species Information

   AIR
   Molecular Weight (g/mol)            28.75199
   Ambient Density (kg/m^3)            1.195
   Initial Mass Fraction               1.000
   Enthalpy of Formation (J/kg)     -8.50E+04

   Sub Species                    Mass Fraction     Mole Fraction
   NITROGEN                       7.624701E-01      7.825730E-01
   OXYGEN                         2.309974E-01      2.075589E-01
   CARBON DIOXIDE                 5.914651E-04      3.864120E-04
   WATER VAPOR                    5.940984E-03      9.481679E-03
 
     Viscosity (kg/m/s) Ambient,  293 K:  1.80E-05
                                  500 K:  2.61E-05
                                 1000 K:  4.13E-05
                                 1500 K:  5.37E-05
   Therm. Cond. (W/m/K) Ambient,  293 K:  2.56E-02
                                  500 K:  3.81E-02
                                 1000 K:  6.69E-02
                                 1500 K:  9.25E-02
        Enthalpy (J/kg) Ambient,  293 K: -9.01E+04
                                  500 K:  1.22E+05
                                 1000 K:  6.72E+05
                                 1500 K:  1.27E+06
    Spec. Heat (J/kg/K) Ambient,  293 K:  1.02E+03
                                  500 K:  1.04E+03
                                 1000 K:  1.16E+03
                                 1500 K:  1.23E+03
   Diff. Coeff. (m^2/s) Ambient,  293 K:  1.99E-05
                                  500 K:  4.96E-05
                                 1000 K:  1.58E-04
                                 1500 K:  3.09E-04


 Material Information


 Surface Conditions

   0 INERT (DEFAULT)

   1 OPEN
     Passive Vent to Atmosphere

   2 MIRROR
     Symmetry Plane

   3 INTERPOLATED

   4 PERIODIC

   5 HVAC

   6 MASSLESS TRACER

   7 DROPLET

   8 EVACUATION_OUTFLOW
     Normal Velocity (m/s)         0.000

   9 MASSLESS TARGET

  10 PERIODIC FLOW ONLY


 Cut-off Density and Temperature

    Minimum Temperature:   -10.0 C
    Maximum Temperature:  2726.8 C
    Minimum Density:  1.195E-01 kg/m3
    Maximum Density:  2.663E+00 kg/m3


 Radiation Model Information

   Number of control angles  104
   Time step increment         3
   Angle increment             5
   Theta band N_phi   Solid angle
     1:        4     0.120
     2:       12     0.114
     3:       16     0.127
     4:       20     0.120
     5:       20     0.120
     6:       16     0.127
     7:       12     0.114
     8:        4     0.120
   Using gray gas absorption.
   Mean beam length  0.100 m



 Run Time Diagnostics

       Time Step       1   October 24, 2020  19:51:40
       Step Size:    0.100E+01 s, Total Time:       1.00 s
       Pressure Iterations:      1
       Maximum Velocity Error:  0.15E-03 on Mesh   4 at (   3   2   4)
       Maximum Pressure Error:  0.14E-08 on Mesh   4 at (   3   2   3)
       ---------------------------------------------------------------
       Mesh    1
       Max CFL number:  0.30E+01 at (   5,   0,   8)
       Max divergence:  0.88E-06 at (   1,   1,   2)
       Min divergence: -0.79E-06 at (   3,   3,   3)
       Max VN number:   0.25E-01 at (   2,   2,   2)
       No. of Lagrangian Particles:           540
       Mesh    2
       Max CFL number:  0.30E+01 at (   5,   0,   8)
       Max divergence:  0.73E-06 at (   2,   2,   2)
       Min divergence: -0.77E-06 at (   2,   3,   3)
       Max VN number:   0.25E-01 at (   2,   2,   2)
       No. of Lagrangian Particles:           540
       Mesh    3
       Max CFL number:  0.30E+01 at (   5,   0,   8)
       Max divergence:  0.34E-06 at (   1,   3,   1)
       Min divergence: -0.90E-06 at (   3,   1,   3)
       Max VN number:   0.30E-01 at (   2,   2,   2)
       No. of Lagrangian Particles:           540
       Mesh    4
       Max CFL number:  0.30E+01 at (   5,   0,   8)
       Max divergence:  0.63E-06 at (   2,   3,   1)
       Min divergence: -0.67E-06 at (   2,   1,   3)
       Max VN number:   0.30E-01 at (   2,   2,   3)
       No. of Lagrangian Particles:           540
       Mesh    5
       Max CFL number:  0.30E+01 at (   5,   0,   8)
       Max divergence:  0.90E-06 at (   1,   2,   2)
       Min divergence: -0.73E-06 at (   3,   3,   2)
       Max VN number:   0.21E-01 at (   2,   3,   2)
       No. of Lagrangian Particles:           540
       Mesh    6
       Max CFL number:  0.30E+01 at (   5,   0,   8)
       Max divergence:  0.67E-06 at (   3,   1,   3)
       Min divergence: -0.71E-06 at (   1,   3,   1)
       Max VN number:   0.20E-01 at (   2,   2,   1)
       No. of Lagrangian Particles:           540
       Mesh    7
       Max CFL number:  0.30E+01 at (   5,   0,   8)
       Max divergence:  0.59E-06 at (   1,   3,   2)
       Min divergence: -0.88E-06 at (   3,   1,   1)
       Max VN number:   0.18E-01 at (   3,   2,   2)
       No. of Lagrangian Particles:           540
       Mesh    8
       Max CFL number:  0.30E+01 at (   5,   0,   8)
       Max divergence:  0.84E-06 at (   3,   3,   2)
       Min divergence: -0.41E-06 at (   2,   3,   2)
       Max VN number:   0.16E-01 at (   2,   1,   2)
       No. of Lagrangian Particles:           540



 Time Stepping Wall Clock Time (s):        0.034
 Total Elapsed Wall Clock Time (s):        1.593

STOP: FDS completed successfully (CHID: 000_distribution_cube_full)
